clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesJob.groovy

```diff
@@ -57,6 +58,19 @@ class KubernetesJob implements Job, Serializable {
     }
   }
 
+  @Override
+  Job.InstanceCounts getInstanceCounts() {
+    new Job.InstanceCounts(
+        down: (Integer) instances?.count { it.healthState == HealthState.Down } ?: 0,
+        outOfService: (Integer) instances?.count { it.healthState == HealthState.OutOfService } ?: 0,
+        up: (Integer) instances?.count { it.healthState == HealthState.Up } ?: 0,
+        starting: (Integer) instances?.count { it.healthState == HealthState.Starting } ?: 0,
+        unknown: (Integer) instances?.count { it.healthState == HealthState.Unknown } ?: 0,
+        succeeded: (Integer) instances?.count { it.healthState == HealthState.Failed } ?: 0,
```

Comment:
Might also want to have the order of the named args match the order of the definitions in `InstanceCounts`.